mobile/lib/main.dart

```diff
@@ -57,139 +59,110 @@ class MainPage extends StatelessWidget {
-                            height: 8,
-                            decoration: const BoxDecoration(
-                              color: Colors.grey,
-                              shape: BoxShape.circle,
-                            ),
-                          ),
+                  Padding(
+                    padding: _widgetPadding,
+                    child: ColorShadowedCard2(
+                      shadowColor: context.colors.highlightRed,
+                      footer: DefaultTextStyle(
+                        style: context.textStyles.textSmall.copyWith(
+                          color: context.colors.highlightGreen,
                         ),
-                        Padding(
-                          padding: const EdgeInsets.symmetric(
-                            horizontal: 2,
-                          ),
-                          child: Container(
-                            width: 8,
-                            height: 8,
-                            decoration: const BoxDecoration(
-                              color: Colors.white,
-                              shape: BoxShape.circle,
+                        child: Padding(
+                          padding: const EdgeInsets.all(10),
+                          child: RichText(
+                            text: TextSpan(
+                              children: [
+                                TextSpan(
+                                  text: 'Teraz gramy \n',
+                                  style: context.textStyles.textMedium.copyWith(
+                                    color: context.colors.highlightGreen,
+                                    height: 1.5,
+                                  ),
+                                ),
+                                TextSpan(
+                                  text:
+                                      'Lorem Ipsum', // TODO: Place for the song title
+                                  style: context.textStyles.textSmall.copyWith(
+                                    color: context.colors.highlightGreen,
+                                    height: 1.5,
+                                  ),
+                                ),
+                              ],
                             ),
                           ),
                         ),
-                        Padding(
-                          padding: const EdgeInsets.symmetric(
-                            horizontal: 2,
-                          ),
-                          child: Container(
-                            width: 8,
-                            height: 8,
-                            decoration: const BoxDecoration(
-                              color: Colors.grey,
-                              shape: BoxShape.circle,
-                            ),
-                          ),
+                      ),
+                      child: Container(
+                        padding: const EdgeInsets.all(80),
```

Comment:
This looks very suspicious. In the future try to find a prettier way to center something. Flex layout might come in handy 😉
It's not really the point of this issue so let's leave it like this for the time being
```suggestion
                        // TODO: redo this layout in a prettier way
                        padding: const EdgeInsets.all(80),
```